package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's this for?

Reply:
That's a new package which is required to build Metabase now